src/__tests__/index.test.ts

```
@@ -7,329 +7,402 @@ const mockSyncService = vi.hoisted(() => ({
+import { AzureDevOpsBot } from '../index.js'
 
-describe("AzureDevOpsBot", () => {
-  let originalEnv: typeof process.env;
-  let originalArgv: string[];
-  let consoleSpy: any;
-  let processExitSpy: any;
-  let bot: AzureDevOpsBot;
+describe('AzureDevOpsBot', () => {
+  let originalEnv: typeof process.env
+  let originalArgv: string[]
+  let consoleSpy: any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)